ontology/yaml/resources/ExcelRedstone/entity_types/VAV.yaml

```diff
@@ -0,0 +1,78 @@
+# VAV Type A
+VAV_PDSCV_CO2C_VOCC_ZHM_FAM_HOA_ZCMD_DFVSCO2C:
+  id: "19ec238b-276d-491e-ba7b-3bba98513f06"
+  description: "Zone air CO2 & VOC control VAV. Variable speed CO2 override control for discharge fans on FCU's fed by VAV when CO2 levels increase above setpoint. Hand/Off/Auto operation is provided, with zone occupied command. Zone air relative humidity and fire alarm status are monitored by this unit."
+  is_canonical: true
+  implements:
+  - VAV # Existing
+  - PDSCV # Existing with optional points added for S2 & Missing ID in Github
+  - CO2C # Existing with optional points added for S2
+  - VOCC # Existing with optional points added for S2
+  - ZHM # Existing with optional points added for S2
+  - FAM # New abstract created for S2
+  - HOA # New abstract created for S2
+  - ZCMD # New abstract created for S2
+  - DFVSCO2C # New abstract created for S2
+
+
+# VAV Type S
+VAV_SDBPC_HOA_OPTM:
```

Comment:
I might argue that this is a DMP and not a VAV, since it is not a terminal unit (its an interstitial damper section). What is the argument for this being classified a VAV (beyond just that it was labeled VAV in the drawings)?

Reply:
bump